stl/inc/flat_map

```diff
@@ -335,9 +348,9 @@ private:
         }
     };
 
-    void _Sort_and_dedup() {
+    void _SortAndDedup() {
         auto _Zip_view = _RANGES views::zip(_Data.keys, _Data.values);
-        _STD sort(_Zip_view.begin(), _Zip_view.end(), value_compare(_Key_compare));
+        _RANGES sort(_Zip_view.begin(), _Zip_view.end(), value_compare(_Key_compare));

```

Comment:
by using `ranges::sort` we can do `_RANGES sort(_Zip_view, value_compare(_Key_compare))`.